zerver/lib/utils.py

```diff
@@ -107,3 +107,10 @@ def process_list_in_batches(
             break
         process_batch(items)
         offset += chunk_size
+
+
+def parse_jitsi_server_url(value: str, special_values_map: Mapping[str, str]) -> str:
+    if value in special_values_map.keys():
+        return special_values_map[value]
+
+    return value
```

Comment:
This should go somewhere else; probably just at the top of `zerver/views/video_calls.py` would be best?  

We generally never want to put things into a file named `utils.py` if we can avoid it; they are code smells since it's totally unclear what should be there :)